src/components/BnCheckbox/BnCheckbox.vue

```diff
@@ -30,6 +32,7 @@ const { handleChange, checked, meta, setTouched } = useField(props.name, props.r
   type: 'checkbox',
   checkedValue: props.value,
   initialValue: props.modelValue,
+  uncheckedValue: props.uncheckedValue,
```

Comment:
Wouldn't this have to be a reference?

Reply:
True. I just added a commit fixing this. Checking this https://github.com/logaretm/vee-validate/blob/main/packages/vee-validate/src/Field.ts#L119C1-L123C49 it seems that almost everything that is used in `useField` should be transformed into a ref first.